src/Uno.UI.Runtime.Skia.X11/X11DisplayInformationExtension.cs

```diff
@@ -229,18 +229,16 @@ internal void UpdateDetails()
 
 				var rawScale = _scaleOverride ?? (TryGetXResource(XftDotdpi, out var xrdbScaling) ? xrdbScaling.Value : dpi / DisplayInformation.BaseDpi);
 
-				var flooredScale = FloorScale(rawScale);
-
 				// This returns very incorrect numbers as far as I've tested.
 				var widthInInches = (uint)Math.Round(X11Helper.XWidthMMOfScreen(screen) / InchesToMilliMeters);
 				var heightInInches = (uint)Math.Round(X11Helper.XHeightMMOfScreen(screen) / InchesToMilliMeters);
 
 				_details = new(
 					(uint)X11Helper.XWidthOfScreen(screen),
 					(uint)X11Helper.XHeightOfScreen(screen),
-					flooredScale * DisplayInformation.BaseDpi,
-					flooredScale,
-					(ResolutionScale)(int)(flooredScale * 100.0),
+					(float)(rawScale * DisplayInformation.BaseDpi),
```

Comment:
Why was the flooring needed originally?

Reply:
I don't remember, but I think I was just copying some other target (FB I think). It shouldn't be needed.